src/Microsoft.Win32.Registry/tests/RegistryTestsBase.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit;
+
+namespace Microsoft.Win32.RegistryTests
+{
+    public abstract class RegistryTestsBase : IDisposable
+    {
+        protected readonly string _testRegistryKeyName;
+        protected readonly RegistryKey _testRegistryKey;
+
```

Comment:
Doesn't really matter for tests, but I'd expect PascalCased properties not underscore fields for protected members.